server/auth/init.js

```diff
@@ -1,17 +1,8 @@
 import passport from 'passport';
-import models from '../models';
-
-const userModel = models.User;
 
 const init = () => {
   passport.serializeUser((user, done) => {
-    done(null, user.id);
-  });
-
-  passport.deserializeUser((id, done) => {
-    userModel.findById(id, (err, user) => {
-      done(err, user);
-    });
+    done(null, user);
```

Comment:
I am not sure if removing the deserializeUser is a good idea. Becuase even the mock test uses both serialize and deserialize. So please can you add it back.

Reply:
Thanks for your review and comment @Oluwafayokemi. desirializeUser is only used when you are trying to get the user object signed into session by passport and since we are not using session, it is just redundant and actually affecting our test coverage as it never gets called. Also, the mock test does not actually use it. The mock test as it right now does not actually do anything. It is actually falling back to the catch-all route we defined in server/routes/index.js file (line 17).

Reply:
OK